### PR TITLE
Add calculator components

### PR DESCRIPTION
**Pull request by Movgto**
# Changes:
- Add calculator react components to build the application interface according to the requirements template.
- No functionality implemented yet as requirements suggest for this update.
- Linters tests passing.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        <Button content="-" operator="true" />
+      </div>
+      <div className="row">
+        <Button content="0" operator="true" double="true" />
+        <Button content="." />
+        <Button content="=" operator="true" />
+      </div>
+    </div>
+  );
+};
```
Comment (DeliceLydia) [inline src/components/Calculator.js]: - [ ] This component should be `functional` for a moment

Comment (DeliceLydia) [reply src/components/Calculator.js]: - [ ] Make sure all the boilerplate that got created by the CRA (texts, images, and styles) are deleted.

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @Movgto, 

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Created `calculator` component
- Followed git workflow
- No linter errors


## Required Changes ♻️
_Check the comments under the review._



### Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (Movgto) [reply src/components/Calculator.js]: @DeliceLydia Hi, I now deleted the remaining boilerplate but the components are already of the functional type, I don't understand 😕

Comment (DeliceLydia) [reply src/components/Calculator.js]: @Movgto Currently you used arrow functions but requirements say the component should be functional
like
```
function Calculator() {
}
```

Comment (Movgto) [reply src/components/Calculator.js]: Thank you, I now just changed it to a functional component, do I need to request another review? @DeliceLydia

**Review by Strangeal — APPROVED** [review]
Hi @Movgto, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- Deleted boilerplate ✅ 
- used functional component ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**